authcrypt/authcrypt.cpp

```diff
@@ -41,8 +39,11 @@ const char Authcrypt::message[] = "Some things are better left unread";
 
 const char Authcrypt::metadata[] = "eg sequence number, routing info";
 
-Authcrypt::Authcrypt()
+Authcrypt::Authcrypt(mbedtls_platform_context* platform_ctx)
```

Comment:
Why is it required to pass ```platform_ctx``` to the application(test)?

Reply:
This can be limited to the tests that actually test things from the platform context.